django_filters/filters.py

```diff
@@ -623,7 +623,7 @@ def get_lookup_choices(self):
             field = get_model_field(self.model, self.field_name)
             lookups = field.get_lookups()
 
-        return [self.normalize_lookup(l) for l in lookups]
+        return [self.normalize_lookup(lookup) for lookup in lookups]
```

Comment:
I consider this a noise as it has nothing to do with the rest of the code and the proposal of the PR. I created another PR https://github.com/carltongibson/django-filter/pull/1225 making the same change and it will be better if it gets merged before the current one so that I can rebase and drop the last commit.

Reply:
See https://github.com/carltongibson/django-filter/commit/1ebb03a396182ea96cdc8165022ad277908413fa. You can rebase. Thanks!